docs/ops/filesystems.md

```diff
@@ -102,6 +102,8 @@ fs.<scheme>.limit.input: (number, 0/-1 mean no limit)
 fs.<scheme>.limit.output: (number, 0/-1 mean no limit)
 fs.<scheme>.limit.timeout: (milliseconds, 0 means infinite)
 fs.<scheme>.limit.stream-timeout: (milliseconds, 0 means infinite)
+fs.<scheme>.limit.rateLimitingInput: (bytes/s, 0 means infinite)
```

Comment:
`input-rate-limit`? + document  default value? Or the default value is visible via some automatically generated docs?